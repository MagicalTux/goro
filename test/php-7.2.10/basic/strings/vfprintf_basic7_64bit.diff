     *** Testing vfprintf() : basic functionality - using unsigned format ***
002- 18446744073709550505
003- 18446744073709550505 18446744073708317049
004- 18446744073709550505 18446744073708317049 18446744073707206184
005- ===DONE===
002+ Warning: fopen(): fopen(/home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic7_64bit.txt): failed to open stream: No such file or directory in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic7_64bit.php on line 20
003+ panic: runtime error: invalid memory address or nil pointer dereference
004+ [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6ae1bd]
005+ 
006+ goroutine 1 [running]:
007+ github.com/MagicalTux/goro/core/compiler.(*runReturn).Run(0xc0000ee408, {0x8bd028, 0xc0003ba000})
008+ 	/home/ronald/proj/goro/core/compiler/compile-return.go:38 +0x5d
009+ github.com/MagicalTux/goro/core/compiler.(*runnableIf).Run(0xc0000e1c40, {0x8bd028, 0xc0003ba000})
010+ 	/home/ronald/proj/goro/core/compiler/compile-if.go:28 +0x233
011+ github.com/MagicalTux/goro/core/phpv.Runnables.Run({0xc0000fe000, 0x14, 0x20}, {0x8bd028, 0xc0003ba000})
012+ 	/home/ronald/proj/goro/core/phpv/run.go:9 +0x103
013+ github.com/MagicalTux/goro/core/phpctx.(*Global).Require(0xc0003ba000, {0x8bd028, 0xc0003ba000}, {0x7ffc48e1c712, 0x4e})
014+ 	/home/ronald/proj/goro/core/phpctx/global-inc.go:116 +0x4da
015+ github.com/MagicalTux/goro/core/phpctx.(*Global).RunFile(0xc0003ba000, {0x7ffc48e1c712, 0x4e})
016+ 	/home/ronald/proj/goro/core/phpctx/global.go:246 +0x7f
017+ main.main()
018+ 	/home/ronald/proj/goro/sapi/php-cli/main.go:60 +0x6a9
