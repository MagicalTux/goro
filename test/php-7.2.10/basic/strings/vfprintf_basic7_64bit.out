*** Testing vfprintf() : basic functionality - using unsigned format ***
Warning: fopen(): fopen(/home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic7_64bit.txt): failed to open stream: No such file or directory in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic7_64bit.php on line 20
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6ae1bd]

goroutine 1 [running]:
github.com/MagicalTux/goro/core/compiler.(*runReturn).Run(0xc0000ee408, {0x8bd028, 0xc0003ba000})
	/home/ronald/proj/goro/core/compiler/compile-return.go:38 +0x5d
github.com/MagicalTux/goro/core/compiler.(*runnableIf).Run(0xc0000e1c40, {0x8bd028, 0xc0003ba000})
	/home/ronald/proj/goro/core/compiler/compile-if.go:28 +0x233
github.com/MagicalTux/goro/core/phpv.Runnables.Run({0xc0000fe000, 0x14, 0x20}, {0x8bd028, 0xc0003ba000})
	/home/ronald/proj/goro/core/phpv/run.go:9 +0x103
github.com/MagicalTux/goro/core/phpctx.(*Global).Require(0xc0003ba000, {0x8bd028, 0xc0003ba000}, {0x7ffc48e1c712, 0x4e})
	/home/ronald/proj/goro/core/phpctx/global-inc.go:116 +0x4da
github.com/MagicalTux/goro/core/phpctx.(*Global).RunFile(0xc0003ba000, {0x7ffc48e1c712, 0x4e})
	/home/ronald/proj/goro/core/phpctx/global.go:246 +0x7f
main.main()
	/home/ronald/proj/goro/sapi/php-cli/main.go:60 +0x6a9